metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/OaipmhHarvestPlugin.java

```diff
@@ -32,6 +33,7 @@
   private String id;
   private PluginStatus pluginStatus = PluginStatus.INQUEUE;
   private static final PluginType pluginType = PluginType.OAIPMH_HARVEST;
```

Comment:
Name of this constant should be all caps.

Reply:
That indeed is true. Again json serialization is important, just checked and it works as before because the getter is in place.